cred.json

```diff
@@ -0,0 +1 @@
+{"request_id":"8001f90f-a456-f860-2cfa-62e2eaad52fc","lease_id":"","renewable":false,"lease_duration":604800,"data":{"admin":"admin","adminpasswd":"D6Z83bIgCP47","coredata":"coredata","coredatapasswd":"O6C8lPw3z9X0","logging":"logging","loggingpasswd":"3E6Fi29WP5Xm","metadata":"metadata","metadatapasswd":"6x8Z201GMLyt","notifications":"notifications","notificationspasswd":"8AV4Mfq5ap97","rulesengine":"rules_engine_user","rulesenginepasswd":"Xq378iS5Z9rD","scheduler":"scheduler","schedulerpasswd":"i0YPr4LV18v9"},"wrap_info":null,"warnings":null,"auth":null}
```

Comment:
References to this file elsewhere are all commented out. Is it still required?

Reply:
This is the testing file for parsing the token so I would prefer to keep it.